docs/guides/hedgehog-engine/rangers/materials/advanced-mats.md

```diff
@@ -0,0 +1,91 @@
+---
+description: Advanced Material Editing guide for Sonic Frontiers
+---
+# Advanced Material Editing
+
+This guide will assume you have at least some understanding of how PBR texturing works.
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeEdit](https://drive.google.com/file/d/1VDHTpPe1MNVlAYblLsDd3Oa1KpsLqLGs/view){ target="_blank"}
+    - [Nvidia Texture Tools Exporter](https://developer.nvidia.com/nvidia-texture-tools-exporter){ target="_blank"}
+
+!!! note
+    There is a [Substance Painter Export Template](hhttps://cdn.discordapp.com/attachments/987489054377508924/1044325602452312124/Frontiers_PRM1.spexp){ target="_blank"} if you use Substance to export the PRM map so you don't have to construct it.
+
+## Preparation
+This guide expects you to have a model that loads ingame and 5 texture maps for your model:
+
+- An `Ambient Occlusion` Map *(if you do not want this make a solid `1` texture)*
+- A `Metallic` Map *(if you do not want this make a solid `0` texture)*
+- A `f0 Specular` Map *(if you do not want this make a solid `0.25` texture)*
+- A `Roughness` Map *(if you do not want this make a solid `0.8` texture)*
+- A `DirectX Normal` Map *(if you do not want this make a solid `1, 1, 0.5` texture)*
+
+![](./assets/01-textures.png)
```

Comment:
Assets should be stored in a folder that has the same name as the .md file. So in this case:

```
./assets/advanced-mats/file1.png
./assets/advanced-mats/file2.png
./assets/advanced-mats/file3.spexp
...
```